pkg/server/api/agent/v1/service.go

```diff
@@ -204,7 +211,149 @@ func (s *Service) AttestAgent(stream agent.Agent_AttestAgentServer) error {
 }
 
 func (s *Service) RenewAgent(stream agent.Agent_RenewAgentServer) error {
-	return status.Error(codes.Unimplemented, "method not implemented")
+	ctx, cancel := context.WithCancel(stream.Context())
```

Comment:
What is this for? If it's to enforce stream cancellation, gRPC will already take care of that when this handler returns.